### PR TITLE
Pile of Minor Fixes

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======

Various small fixes for the final tweaks of the docs

Closes: #87

Solution
========

- Updated print styles
- Updated all the colors to reflect variables from the style guide and consistency
- Fixed side nav issue with hiding more than it should and margins
    - image
- Updated button styles
    - image
- Shifted the selected header indicator `#` to be visible with the new header font
    - image
- Non breaking space for the arrows so they will break better
    - image

### CODE REVIEW

File: css/button-links.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.button-links,
```
Comment (wilwade) [inline css/button-links.css:1]: Separated out for easier handling

File: css/button-links.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.button-links a:hover,
+.button-links > p a:hover,
+.button-links a:visited:hover,
+.button-links > p a:visited:hover {
+  text-decoration: none;
+  border: 2px solid var(--button-nav-hover-border);
+  color: var(--button-nav-hover-fg);
+  background: var(--button-nav-hover-bg);
```
Comment (wilwade) [inline css/button-links.css:73]: Cleaned up these variables

File: css/side-nav.css [hunk 3]
```diff
@@ -46,25 +46,25 @@
 }
 
 /* Individual icons for specific items (1 indexed) */
-#toc > .chapter li.chapter-item:nth-child(2) > a:first-of-type::before {
+#toc > .chapter > li.chapter-item:nth-child(2) > a:first-of-type::before {
```
Comment (wilwade) [inline css/side-nav.css:49]: These were only supposed to catch the icon ones.

File: css/side-nav.css [hunk 4]
```diff
@@ -84,21 +84,25 @@
...
-#toc > .chapter li.chapter-item > a:first-of-type {
+#toc > .chapter > li.chapter-item > a:first-of-type {
   padding-inline-start: 0;
   display: flex;
   align-items: center;
   flex: 1;
 }
 
+#toc > .chapter > li > .section > li.chapter-item:first-of-type {
+  margin-top: 10px;
```
Comment (wilwade) [inline css/side-nav.css:103]: Fix the margin under the first one under the icons (Aka just Core > Arch for now)

**Review by aramikm — APPROVED** [review]

File: theme/css/chrome.css [hunk 5]
```diff
@@ -376,6 +385,7 @@ mark {
   margin-block-end: -1px;
   margin-inline-start: -3px;
   margin-inline-end: -3px;
+  color: var(--fg);
```
Comment (wilwade) [inline theme/css/chrome.css:388]: Search text color fix

File: theme/css/variables.css [hunk 6]
```diff
@@ -11,28 +11,57 @@
   --mono-font: "Source Code Pro", Consolas, "Ubuntu Mono", Menlo, "DejaVu Sans Mono", monospace, monospace;
   --code-font-size: 0.875em /* please adjust the ace font size accordingly in editor.js */;
   --header-height: 85px;
+
+  /* Colors from Style Guide */
```
Comment (wilwade) [inline theme/css/variables.css:15]: Pulled these in and used them to keep things straight(er)

**Pull request merged**